src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,11 @@
+<arquillian xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns="http://jboss.org/schema/arquillian"
+            xsi:schemaLocation="http://jboss.org/schema/arquillian
+               http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+
+  <extension qualifier="openshift">
+    <property name="namespace.use.current">true</property>
```

Comment:
Can't we configure these from a mavne project (system properties) and drop the arquillian.xml?

Reply:
I'm not sure if I follow, you can still override them with CLI maven params, like `mvn clean verify -Popenshift,openshift-it -Dnamespace.use.current=false`

And yes, they can also be moved to `pom.xml` and be passed to surefire plugin configuration as system property variables so you don't have to have the `arquillian.xml` file, but isn't it a bit more readable this way ?